app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Defining `skills` as a class attribute means it will be shared across all instances of `SoftwareEngineer`. Consider moving `skills` to the constructor (`__init__` method) as an instance attribute to ensure each instance has its own list of skills.